### PR TITLE
[INLONG-5228][Manager] Support create SQLServer databases and tables

### LINKED ISSUE #5228: [Feature][Manager] Support create SQLServer databases and tables
### Description

To support the creation of MySQL databases and tables, you can refer to the class HiveResourceOperator.

### Use case

No response

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by haibo-duan**
### Prepare a Pull Request
*(Change the title refer to the following example)*
Support create SqlServer databases and tables

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #5228 

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve?*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

**Review by woofyzhao — APPROVED** [review]

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sqlserver/SQLServerJdbcUtils.java [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+     *
+     * @param conn JDBC Connection  {@link Connection}
+     * @param schemaName SqlServer schema name
+     * @throws Exception on create schema error
+     */
+    public static void createSchema(Connection conn, String schemaName) throws Exception {
+        if (!checkSchemaExist(conn, schemaName)) {
+            String createSql = SQLServerSqlBuilder.buildCreateSchemaSql(schemaName);
+            executeSql(conn, createSql);
+            LOG.info("execute sql [{}] success! ", createSql);
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sqlserver/SQLServerJdbcUtils.java:137]: No need to add blank or punctuation at the end of log info.

**Review by healchow — APPROVED** [review]
LGTM

**Comment by healchow** [comment]
Please fix the compile error first, thanks.

File: inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/core/sink/SQLServerSinkServiceTest.java [hunk 2]
```diff
@@ -98,4 +106,68 @@ public void testGetAndUpdate() {
...
+     * @param tableName SqlServer table name
+     * @return {@link SQLServerTableInfo}
+     */
+    private final SQLServerTableInfo bulidTableInfo(String schemaName, String tableName) {
+        SQLServerTableInfo tableInfo = new SQLServerTableInfo();
+        tableInfo.setTableName(tableName);
+        tableInfo.setComment("test01 ");
+        tableInfo.setPrimaryKey("id");
+        tableInfo.setSchemaName(schemaName);
+        List<SQLServerColumnInfo> columnInfos = new ArrayList<>();
```
Comment (leosanqing) [inline inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/core/sink/SQLServerSinkServiceTest.java:161]: Initializing columnInfos should be separated from the field logic of set tableInfo, with a blank line in the middle, which will be more readable

File: inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/core/sink/SQLServerSinkServiceTest.java [hunk 3]
```diff
@@ -98,4 +106,68 @@ public void testGetAndUpdate() {
...
+        tableInfo.setColumns(columnInfos);
+        SQLServerColumnInfo id = new SQLServerColumnInfo("id", "int", "id");
+        columnInfos.add(id);
+        SQLServerColumnInfo cell = new SQLServerColumnInfo("cell", "varchar(20)", "cell");
+        columnInfos.add(cell);
+        SQLServerColumnInfo name = new SQLServerColumnInfo("name", "varchar(40)", "name");
+        columnInfos.add(name);
+        tableInfo.setPrimaryKey("id");
```
Comment (leosanqing) [inline inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/core/sink/SQLServerSinkServiceTest.java:169]: Personally, the add operation can be combined into one sentence using Guava's Lists.new ArrayList().
```
List<SQLServerColumnInfo> columnInfos = Lists.newArrayList(
                new SQLServerColumnInfo("id", "int", "id"),
                new SQLServerColumnInfo("cell", "varchar(20)", "cell"),
                new SQLServerColumnInfo("name", "varchar(40)", "name")
        )
```

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/sqlserver/SQLServerSqlBuilder.java [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+                    .append(" ;");
+        }
+        sqls.add(sqlBuilder.toString());
+        columnList.stream().filter(column -> StringUtils.isNotEmpty(column.getComment()))
+                .forEach(column -> {
+                    sqls.add(
+                            buildAddColumnComment(schemaName, tableName, column.getName(), column.getComment())
+                    );
+                });
+        LOGGER.info("add columns sql={}", sqls);
```
Comment (leosanqing) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/sqlserver/SQLServerSqlBuilder.java:163]: This one will only print the address of the List not the element

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/sqlserver/SQLServerSqlBuilder.java [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+        StringBuilder sql = new StringBuilder();
+        // Support _ beginning with underscore
+        sql.append("CREATE TABLE ").append(table.getSchemaName())
+                .append(".")
+                .append(table.getTableName());
+
+        // Construct columns and partition columns
+        sql.append(buildCreateColumnsSql(table));
+        sqls.add(sql.toString());
+        table.getColumns().stream().filter(column -> StringUtils.isNotEmpty(column.getComment()))
```
Comment (leosanqing) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/sqlserver/SQLServerSqlBuilder.java:124]: Personally, I think it is better to change the filter operation to a new line.

Comment (haibo-duan) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/sqlserver/SQLServerSqlBuilder.java:124]: OK

File: inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/core/sink/SQLServerSinkServiceTest.java (hunk 3 above)
Comment (haibo-duan) [reply inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/core/sink/SQLServerSinkServiceTest.java:169]: its a good idea

**Review by dockerzhang — APPROVED** [review]

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/sqlserver/SQLServerSqlBuilder.java (hunk 4 above)
Comment (leosanqing) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/sqlserver/SQLServerSqlBuilder.java:162]: My bad, pls ignore this

**Review by leosanqing — APPROVED** [review]

**Pull request merged**